src/main/kotlin/no/nav/tilleggsstonader/klage/behandling/domain/Behandling.kt

```diff
@@ -48,18 +52,11 @@ data class PåklagetVedtak(
 
 enum class PåklagetVedtakstype {
     VEDTAK,
-    INFOTRYGD_TILBAKEKREVING,
+    TILBAKEKREVING,
     UTEN_VEDTAK,
     IKKE_VALGT,
-    UTESTENGELSE,
-    INFOTRYGD_ORDINÆRT_VEDTAK,
```

Comment:
Er tanken å fjerne funksjonalitet nå, for å senere eks legge til støtte for å kunne klage på arena-vedtak?